aws/resource_aws_api_gateway_resource.go

```diff
@@ -60,9 +60,8 @@ func resourceAwsApiGatewayResourceCreate(d *schema.ResourceData, meta interface{
 	}
 
 	d.SetId(*resource.Id)
-	d.Set("path", resource.Path)
 
-	return nil
+	return resourceAwsApiGatewayResourceRead(d, meta)
```

Comment:
Was there any specific reason for this change - any bug or misbehaviour you experienced or is this just to stick to conventions?